### PR TITLE
Change icon-messages in header to default

### PR DESCRIPTION
**Pull request by rudenkoserhii**
## Describe your changes

- Changed icon unread messages to template default
- Hide notification center (in future will work fully)
- Add missed locales on main-page
- Change query

Screenshot 2024-01-05 165509 The circle around mailbox it's hover effect
Screenshot 2024-01-05 165555

## Issue ticket code (and/or) and link

- [Link to JIRA ticket](https://testhomework4.atlassian.net/browse/AG-210?atlOrigin=eyJpIjoiZDA0MzRmODhmZjk0NDcxYThmYjczOTZmYzhhZDg1MmIiLCJwIjoiaiJ9)

### **General**

- [x] Assigned myself to the PR
- [x] Assigned the appropriate labels to the PR
- [x] Assigned the appropriate reviewers to the PR
- [ ] Updated the documentation
- [x] Performed a self-review of my code
- [x] Types for input and output parameters
- [x] Don't have "any" on my code
- [x] Used the try/catch pattern for error handling
- [x] Don't have magic numbers
- [x] Compare only with constants not with strings
- [x] No ternary operator inside the ternary operator
- [x] Don't have commented code
- [x] no links in the code, env links should be in env file (for example: server url), constant links (for example default avatar URL) should be in constant file.
- [x] Used camelCase for variables and functions
- [x] Date and time formats are on the constants
- [x] Functions are public only if it's used outside the class
- [x] No hardcoded values
- [ ] covered by tests
- [x] Check your commit messages meet the [conventional commit format](https://www.conventionalcommits.org/en/v1.0.0/).

### Frontend

- [x] Components and business logic are separated
- [x] Colors, Font Size, and Font Name is on the theme or in the constants
- [x] No text in the components, use i18n approach
- [x] No inline styles
- [x] Imports are absolute
- [x] Attach a screenshot if PR has visual changes.

### CODE REVIEW

**Review by canterbery — APPROVED** [review]

**Comment by lyudad** [comment]
Screenshot 2024-01-04 at 22 10 08

File: src/locales/langs/en.json [hunk 1]
```diff
@@ -350,6 +350,10 @@
     "forgotPass": "Forgot password?"
   },
   "mainPage": {
+    "My properties": "My properties",
```
Comment (lyudad) [inline src/locales/langs/en.json]: почему в ключах есть пробелы и почему ключи с большой буквы

Comment (rudenkoserhii) [reply src/locales/langs/en.json]: Fixed

File: src/app/[locale]/leads/lib/components/leads-filter/leads-filter.tsx [hunk 2]
```diff
@@ -41,7 +41,10 @@ function LeadsFilter({ getFilter }: Prop): JSX.Element {
       status: data.status === randomValues.ALL ? '' : data.status,
     };
 
-    const searchString = `ContactNameContacts.name contains '${modifiedData.leadName}' and Opportunities.conversion_status matches '${modifiedData.status}'`;
+    const searchString =
+      modifiedData.status === '' && modifiedData.leadName === ''
```
Comment (lyudad) [inline src/app/[locale]/leads/lib/components/leads-filter/leads-filter.tsx:45]: разве !modifiedData.status || !modifiedData.leadName не тоже самое?

File: src/app/[locale]/leads/lib/components/leads-filter/leads-filter.tsx [hunk 3]
```diff
@@ -41,7 +41,10 @@ function LeadsFilter({ getFilter }: Prop): JSX.Element {
...
+        ? 'update'
```
Comment (lyudad) [inline src/app/[locale]/leads/lib/components/leads-filter/leads-filter.tsx:46]: это текст? или переменная ? мне не нравится что она строкой

File: src/app/[locale]/leads/lib/components/leads-filter/leads-filter.tsx (hunk 2 above)
Comment (rudenkoserhii) [reply src/app/[locale]/leads/lib/components/leads-filter/leads-filter.tsx:45]: Was not my changes

File: src/app/[locale]/leads/lib/components/leads-filter/leads-filter.tsx (hunk 3 above)
Comment (rudenkoserhii) [reply src/app/[locale]/leads/lib/components/leads-filter/leads-filter.tsx:46]: Was not my changes

**Pull request merged**